From acd673711f3b32e714226ea563d8d7f0afff7251 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Sat, 16 Nov 2013 00:32:30 +0100
Subject: [PATCH 09/10] Always build uic

Even if we are not building gui or widgets. This tool is needed later
as a native tool when compiling the target.

Change-Id: I257668ac28c22b192e7ec7736e6c23fa3be6bab6
Signed-off-by: Mikko Levonmaa <mikko.levonmaa@palm.com>
Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 src/src.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/src.pro b/src/src.pro
index 1671338..83de483 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -180,7 +180,7 @@ contains(QT_CONFIG, concurrent):SUBDIRS += src_concurrent
         }
     }
 }
-SUBDIRS += src_plugins
+SUBDIRS += src_plugins src_tools_uic
 
 nacl: SUBDIRS -= src_network src_testlib
 
-- 
2.7.1

